tests/utils/gossamer_utils.go

```diff
@@ -447,10 +448,12 @@ func generateDefaultConfig() *ctoml.Config {
 			NoMDNS:      false,
 		},
 		RPC: ctoml.RPCConfig{
-			Enabled:   false,
-			Host:      "localhost",
-			Modules:   []string{"system", "author", "chain", "state"},
-			WSEnabled: false,
+			Enabled:           false,
+			ExternalEnabled:   false,
+			Host:              "localhost",
+			Modules:           []string{"system", "author", "chain", "state"},
+			WSEnabled:         false,
+			WSExternalEnabled: false,
```

Comment:
same here, this can stay as before, the integration tests don't use ws at the moment